api/v1beta2/foundationdbcluster_types.go

```diff
@@ -1215,6 +1215,12 @@ type AutomaticReplacementOptions struct {
 	// The default is false.
 	Enabled *bool `json:"enabled,omitempty"`
 
+	// FaultDomainBasedReplacements controls whether automatic replacements are targeting all failed process groups
+	// in a fault domain or only specific Process Groups. If this setting is enabled MaxConcurrentReplacements will
+	// provide the information in how many fault domains failed process groups can be replaced.
+	// The default is true.
+	FaultDomainBasedReplacements *bool `json:"faultDomainBasedReplacements,omitempty"`
```

Comment:
We have to run `make clean all` to generate the new docs and we have to update the according docs.